### PR TITLE
Prevent exception when using deleteList on a single record

### PR DESCRIPTION
**Pull request by jcroteau**

### CODE REVIEW

File: lib/netsuite/actions/delete_list.rb [hunk 1]
```diff
@@ -58,7 +58,17 @@ def request_body
...
       def response_list
-        @response_list ||= @response.to_hash[:delete_list_response][:write_response_list][:write_response]
+        @response_list ||= array_wrap(@response.to_hash[:delete_list_response][:write_response_list][:write_response])
+      end
+
+      def array_wrap(object)
+        if object.nil?
+          []
+        elsif object.respond_to?(:to_ary)
+          object.to_ary || [object]
```
Comment (greggroth) [inline lib/netsuite/actions/delete_list.rb]: Do you have an example of when `object.to_ary` would return something falsey?

Comment (jcroteau) [inline lib/netsuite/actions/delete_list.rb]: I just copied this from the ```Array.wrap``` extension provided by Rails, so it's meant for a much more general use case than it is here. It's possible, though perhaps not probable, that an object can have ```to_ary``` defined to return ```nil```. I think the intention of this method as written is to guarantee that the return value is an Array under all circumstances, including this one.

**Comment by iloveitaly** [comment]
@jcroteau Thanks for your work here! I ended up adding a `array_wrap` to `Support::Requests` and fixing this bug for `get_all` and `delete_list`

2874194...528007b

**Comment by jcroteau** [comment]
@iloveitaly glad to see it came in handy :)

**Pull request closed without merging**